src/OpenTelemetry.Exporter.Geneva/MessagePackSerializer.cs

```diff
@@ -571,4 +578,14 @@ public static int Serialize(byte[] buffer, int cursor, object obj)
                 return SerializeUnicodeString(buffer, cursor, repr);
         }
     }
+
+    public static int SerializeSpan(byte[] buffer, int cursor, Span<byte> span, int size)
```

Comment:
This is a very specific "Span" of the type byte. Choose the method name to reflect that. `SerializeByteSpan`?

Reply:
I'm not sure whether rename it to be `SerializeByteSpan` would add much value for the reader.
Some thoughts I have in mind:
1. Users are able to tell the type of 3rd parameter is `Span<byte>` from the caller side.
2. Not including the param type is also the current naming convention we have in https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/faaa8b56eae0196754ec1162d4cc7a8253d45baa/src/OpenTelemetry.Exporter.Geneva/MessagePackSerializer.cs#L472

Reply:
I was looking at one of the other existing methods in the class called `SerializeArray`. The method name does not have the type name in it but that's because this method is generic and is used to serialize an array of any type `T[]`:

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/faaa8b56eae0196754ec1162d4cc7a8253d45baa/src/OpenTelemetry.Exporter.Geneva/MessagePackSerializer.cs#L434

Having said that, I'm okay with this:

> 1. Users are able to tell the type of 3rd parameter is `Span<byte>` from the caller side.